cfgov/v1/models/filterable_list_mixins.py

```diff
@@ -95,7 +97,11 @@ def get_context(self, request, *args, **kwargs):
 
         form_data, has_active_filters = self.get_form_data(request.GET)
         queryset = self.get_filterable_queryset()
-        has_archived_posts = queryset.filter(is_archived='yes').count() > 0
+        # flag check to enable or disable archive filter options
+        if flag_enabled('ARCHIVE_FILTER_OPTIONS', request=request):
+            has_archived_posts = False
+        else:
+            has_archived_posts = queryset.filter(is_archived='yes').count() > 0
```

Comment:
I suspect to pass the coverage test, you'll need to copy the test that covers this method with `@override_settings` to enable the flag, as well as with the flag disabled.

Reply:
Added `HIDE_ARCHIVE_FILTER_OPTIONS` flag to `v1/tests/models/test_filterable_list_mixins.py`